src/SDL2.cs

```diff
@@ -3961,6 +3961,91 @@ public struct SDL_Color
 			public byte g;
 			public byte b;
 			public byte a;
+
+			public SDL_Color() { r = 0; g = 0; b = 0; a = 0; }
+			public SDL_Color(byte r, byte g, byte b)
+			{
+				this.r = r;
+				this.g = g;
+				this.b = b;
+				this.a = 255;
+			}
+			public SDL_Color(byte a, byte r, byte g, byte b)
+			{
+				this.r = r;
+				this.g = g;
+				this.b = b;
+				this.a = a;
+			}
+			public SDL_Color(byte a, SDL_Color color)
+			{
+				this.r = color.r;
+				this.g = color.g;
+				this.b = color.b;
+				this.a = a;
+			}
+
+			public SDL_Color(string hex)
```

Comment:
This goes too deep into being custom code (not to mention it's string manipulation which is always a black hole for long-term maintenance).